### PR TITLE
feat: change the position of bar y-axis

### PR DESCRIPTION
**Pull request by owlas**
https://github.com/user-attachments/assets/5fae1e6a-2be3-4dc8-8d31-f5ce44880d6c

### CODE REVIEW

File: packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx [hunk 1]
```diff
@@ -46,12 +55,25 @@ export const BarChartStyling = () => {
...
+                            onYAxisPositionChange(value || undefined)
+                        }
+                        placeholder={'Change the y-axis position'}
+                    />
+                </Config.Section>
+            </Config>
+            <Config>
+                <Config.Section>
+                    <Config.Heading>{`Series`}</Config.Heading>
+                    <TextInput radius="md" />
```
Comment (joaoviana) [inline packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx:76]: do you want to keep this here? The `series` bit

File: packages/common/src/visualizations/SqlRunnerResultsTransformer.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export type BarChartDisplay = {
     };
     yAxis?: {
         label?: string;
-        position?: 'left' | 'right';
+        position?: string;
```
Comment (joaoviana) [inline packages/common/src/visualizations/SqlRunnerResultsTransformer.ts:52]: is this so we don't have an `echarts`-specific type?

**Review by joaoviana — APPROVED** [review]

File: packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx [hunk 3]
```diff
@@ -1,8 +1,11 @@
-import { XLayoutType } from '@lightdash/common';
```
Comment (joaoviana) [inline packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx:1]: Needs this import back so `onXAxisLabelChange` can use it 👍

**Review by joaoviana — CHANGES_REQUESTED** [review]

File: packages/common/src/visualizations/SqlRunnerResultsTransformer.ts (hunk 2 above)
Comment (owlas) [reply packages/common/src/visualizations/SqlRunnerResultsTransformer.ts:52]: It was unnecessarily strict, it meant having to type a lot of other places. I'm experimenting with more relaxed typing, this led to much less boilerplate ❤️

Comment (owlas) [reply packages/common/src/visualizations/SqlRunnerResultsTransformer.ts:52]: e.g. the value from a `Select` `onChange` is going to be a string, unless we do some type gymnastics

File: packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx (hunk 1 above)
Comment (owlas) [reply packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx:76]: I'll remove this!

File: packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx (hunk 3 above)
Comment (owlas) [reply packages/frontend/src/features/sqlRunner/components/BarChartStyling.tsx:1]: on it

**Review by owlas — COMMENTED** [review]
making changes adn merging

**Review by joaoviana — APPROVED** [review]

**Pull request merged**